ivy/functional/ivy/set.py

```diff
@@ -144,26 +144,91 @@ def unique_values(
+           considered distinct, and the corresponding unique element will be
+           implementation-dependent (e.g., an implementation could choose to return
+           ``-0`` if ``-0`` occurs before ``+0``).
 
     Parameters
     ----------
     x
-        input array. If x has more than one dimension, the function must flatten x and
+        input array. If ``x`` has more than one dimension, the function must flatten ``x`` and
         return the unique elements of the flattened array.
 
     Returns
     -------
     ret
-        a namedtuple (values, counts) whose
+        a namedtuple ``(values, counts)`` whose
+
+        - first element must have the field name ``values`` and must be an array containing the
+          unique elements of ``x``.The array must have the same data type as ``x``.
+        - second element must have the field name ``counts`` and must be an array containing
+          the number of times each unique element occurs in ``x``. The returned array must
+          have same shape as ``values`` and must have the default array index data type.
+
+    .. note::
+           The order of unique elements is not specified and may vary between
+           implementations.
+
+    This method conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.elementwise_functions.tan.html>`_
+    in the standard. The descriptions above assume an array input for simplicity, but
+    the method also accepts :code:`ivy.Container` instances in place of
+    :code:`ivy.Array` or :code:`ivy.NativeArray` instances, as shown in the type hints
+    and also the examples below.
+
+    Functional Examples
+    -------------------
+
+    With :code: 'ivy.Array' input:
+
+    >>> x = ivy.array([1,2,1,3,4,1,3])
+    >>> y = unique_counts(x)
+    >>> print(y)
+    Tuple([1,2,3,4],[3,1,2,1])
+
+    >>> x = ivy.asarray([1,2,3,4],[2,3,4,5],[3,4,5,6])
+    >>> y = unique_counts(x)
+    >>> print(y)
+    Tuple([1,2,3,4,5,6],[1,2,3,3,2,1])
+
+    With :code: 'ivy.NativeArray' input:
+
+    >>> x = ivy.NativeArray([0.2,0.3,0.4,0.2,1.4,2.3,0.2])
+    >>> y = ivy.unique_counts(x)
+    >>> print(y)
+    Tuple([0.2,0.3,0.4,1.4,2.3],[3,1,1,1,1]
 
-        - first element must have the field name values and must be an array containing
-          the unique elements of x. The array must have the same data type as x.
-        - second element must have the field name counts and must be an array containing
-          the number of times each unique element occurs in x. The returned array must
-          have same shape as values and must have the default array index data type.
+    Instance Method Examples
+    ------------------------
 
+    Using :code:`ivy.Array` instance method:
+    >>> x = ivy.Array([0., 1., 3. , 2. , 1. , 0.])
```

Comment:
Could u create the array using `ivy.array`

Reply:
Perfect just made those changes now.
Sorry that took a while I'm currently flying back to the UK from India :))